XMonad/Config/Dmwit.hs

```diff
@@ -19,6 +16,7 @@ import System.Process
 
 -- xmonad core
 import XMonad
+import XMonad.Prelude
```

Comment:
This is part of `xmonad-contrib` and perhaps shouldn't be part of a block whose comment is "xmonad core".